### DISCUSSION TITLE
Skeleton Proposal for SASS

### DISCUSSION THREAD

**Discussion by njsch** [eduvis/eduvis.github.io#17] [Q&A]
@Coedice, just wondered what you're thoughts were on the following [gist suggestion](https://gist.githubusercontent.com/njsch/33d5aa901c8bf0131d60539ab71e9be2/raw/162473a5616532f014c0d3519001af6a4b5468dc/eduvis-github-io_sass.scss) for `assets/css/styles.scss`? I have not proposed anything for the `_sass` dir. as I don't know enough about how it works.

Essentially, for low-vision readers, I think we should possibly stick to the recommendations in the comment near the top of the file.

**Comment by Coedice** [comment]
Hi, yes I think that idea makes sense. If you'd like to raise a PR, that can serve basis point for my later and more comprehensive style upgrades. Once there is a PR, I can run it locally and see what that looks like
